settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,60 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: cloud
+              options:
+                  cloud:
+                      type: Value
+                      help: The cloud which the OpenStack modules will operate against. Cloud setup instructions: http://docs.openstack.org/developer/os-client-config/#config-files
+                      required: yes
```

Comment:
Please remove "required" and add to help that "you can source valid keystonerc files instead or define OS_* envvars instead"